7forecast.tex

```diff
@@ -56,44 +55,43 @@ \subsection{Forecast File Options (forecast.ss)}
-    & -0.25 & \Bstrut\\
+  \multicolumn{2}{l}{COND: \hyperlink{Bmarks_RefPoints}{Benchmarks} = 3} & \multirow{1}{1cm}[-0.15cm]{\parbox{11.3cm}{B\textsubscript{LIMIT} as a fraction of the B\textsubscript{MSY} where a negative value will be applied as a fraction of B0}} \Tstrut\\
+    & -0.25 & \\
     
     
   \hline
-  0 0 0 0 0 0 0 0 0 0 & Benchmark Years: & \multirow{1}{1cm}[-0.15cm]{\parbox{11cm}{Requires 10 values, beginning and ending years for (1,2) biology (e.g., growth, natural mortality, maturity, fecundity), (3,4) selectivity, (5,6) relative Fs, (7,8) movement and recruitment distribution; (9,10) stock-recruitment parameters for averaging years in calculating benchmark quantities. If there is no time-varying biology it is recommend to select the first model year for the beginning year for biology.}} \Tstrut\\
+  \multirow{1}{1cm}[-0.15cm]{\parbox{2.4cm}{0 0 0 0 0 0 0 0 0 0}} & Benchmark Years: & \multirow{1}{1cm}[-0.15cm]{\parbox{11.3cm}{Requires 10 values, beginning and ending years for (1,2) biology (e.g., growth, natural mortality, maturity, fecundity), (3,4) selectivity, (5,6) relative Fs, (7,8) movement and recruitment distribution; (9,10) stock-recruitment parameters for averaging years in calculating benchmark quantities. The first model year for the beginning year for biology should be selected if there is no time-varying biology.}} \Tstrut\\
     & -999: start year; & \\
     & >0: absolute year; and & \\
     & <= 0: year relative to end year. & \\
     & & \\
-    & & \\
+ %   & & \\
 
-  \pagebreak
-  %\hline
-  1 & Benchmark Relative F Basis: & \multirow{1}{1cm}[-0.2cm]{\parbox{11cm}{The specification does not affect year range for selectivity and biology.}} \Tstrut\\
+  % \pagebreak
+  \hline
+  1 & Benchmark Relative F Basis: & \multirow{1}{1cm}[-0.2cm]{\parbox{11.3cm}{The specification does not affect year range for selectivity and biology.}} \Tstrut\\
     & 1 = use year range; and & \\
-    & 2 = set range for relF same as forecast below. & \\
+    & 2 = set range for relF same as \hyperlink{Fcast}{Forecast}. & \Bstrut\\
 
   %\pagebreak
   \hline
-  2 & Forecast: & \multirow{1}{1cm}[-0.25cm]{\parbox{11cm}{This input is required but is ignored if benchmarks are turned off. This determines how forecast catches are calculated and removed from the population which is separate from the ``MSY Method'' above. If F\textsubscript{MSY} is selected, it uses whatever proxy (e.g., F\textsubscript{SPR} or F\textsubscript{BTGT}) is selected in the ``MSY Method'' row.}} \Tstrut\\
+  2 & \hypertarget{Fcast}{Forecast:} & \multirow{1}{1cm}[-0.25cm]{\parbox{11.3cm}{This input is required but is ignored if benchmarks are turned off. This determines how forecast catches are calculated and removed from the population which is separate from the ``MSY Method'' above. If F\textsubscript{MSY} is selected, it uses whatever proxy (e.g., F\textsubscript{SPR} or F\textsubscript{BTGT}) is selected in the ``MSY Method'' row.}} \Tstrut\\
```

Comment:
This comment doesn't matter at all but I would put the colon on the outside of the link.